azure/petctl.py

```diff
@@ -0,0 +1,214 @@
+import argparse
+import os
+import subprocess
+import yaml
+import json
+
+# +
+def configure_yaml(yaml_file, petctl_dir,args):
+    
+    print('Configuring azure-pytorch-elastic.yaml....')
+    
+    with open(yaml_file) as f:
+        data = yaml.load(f, Loader=yaml.FullLoader)
+    
+    data["spec"]["parallelism"] = args.max_size
+    data["spec"]["template"]["spec"]["containers"][0]["env"].extend([dict([('name', 'JOB_ID'), ('value', args.name)]),
```

Comment:
We probably want to add some auto-generated suffix to the name to create JOB_ID, otherwise re-running will cause problems because of old info in etcd.